website/docs/addon_usd_artist_get_started.md

```diff
@@ -0,0 +1,98 @@
+---
+id: addon_usd_artist_get_started
+title: Getting Started
+sidebar_label: Getting Started
+---
+
+:::info
+This page is still under construction 👷🚧🛠️🔜🏗️.
+:::
+
+# Getting started
+
+## Creating an asset
+
+The producer has his 🔫 pointing at you and needs the **asset** _now!_
+
```

Comment:
I would basically follow the Maya user guide in structure and language (no cowboys, serious tone instead pls). Even tho its funny its a bit unprofessional imho. It worked for demos and community forums well but no need to use it here.

The maya section is also good for reference as it guides users through quite well step by step and is easy to follow too...

Reply:
Yes good note - I may make this more professional and to the point.

Reply:
maybe just using AI copilot for changing tone? :)